astropy/coordinates/angles.py

```diff
@@ -533,7 +533,7 @@ def __setitem__(self, item, value):
 
     # Any calculation should drop to Angle
     def __array_wrap__(self, obj, context=None):
-        obj = super(Angle, self).__array_wrap__(obj, context=context)
```

Comment:
Okay, I thought that was intentional because the comment said `Any calculation should drop to Angle`. 😄

Reply:
Yes, but `Angle` doesn't have its own `__array_wrap__` -- clearly a mistake on my part!